### DISCUSSION TITLE
Channel

### DISCUSSION THREAD

**Discussion by lan956** [ACINQ/phoenix#340] [Q&A]
Hello, just installed and opened channel on Phoenix ios app. I wonder if I can open another channel/wallet, and can I increase channel size? Thank you.

**Comment by dpad85** [comment]
Hi, sorry I missed you question. Late is better than never, so: the current version of Phoenix opens new channels when needed, that's done automatically. We are working on a update which will allow Phoenix to instead grow/shrink channels with usage. It means you'll be able to only have one single channel, which is much better (for example, no need to split payments).

You can look at this PR to monitor the advancement of this update: #356
